install.sh

```diff
@@ -32,22 +32,39 @@ fi
 
 # Test if Git is available on the Host
 if ! hash git 2>/dev/null; then
-  >&2 echo "Error: Unable to find git in your PATH."
+  echo >&2 "Error: Unable to find git in your PATH."
   exit 1
 fi
 
 # Clone the Shorebird repository into the install_dir
 echo "Cloning Shorebird into $(install_dir)"
 git clone https://github.com/shorebirdtech/shorebird.git -b stable "$(install_dir)"
 
+# Build Shorebird
+eval $(install_dir)/bin/shorebird --version
 
+RELOAD_REQUIRED=false
 SHOREBIRD_BIN="$(install_dir)/bin"
-echo "Checking if $SHOREBIRD_BIN is in your PATH"
-case :$PATH:
-   in *:$SHOREBIRD_BIN:*) ;; # do nothing, it's there
-     *) add_shorebird_to_path >&2;;
+case :$PATH: in *:$SHOREBIRD_BIN:*) ;; # do nothing, it's there
+*)
+  RELOAD_REQUIRED=true
+  add_shorebird_to_path >&2
+  ;;
 esac
 
-# Build Shorebird
-echo "Building Shorebird"
-shorebird
+echo "\n🐦 Shorebird has been installed!"
+
+if [ "$RELOAD_REQUIRED" = true ]; then
+  echo "
+Close and reopen your terminal to start using Shorebird or run the following command to start using it now:
+  
+  export PATH=\"$(install_dir)/bin:\$PATH\""
```

Comment:
source ~/.zshrc would also work, no?

Reply:
Yup I just wanted something a bit more generic instead of assuming zsh

Reply:
👍